test-coverage-dashboard.html

```
@@ -0,0 +1,2038 @@
+                </div>
+            </div>
+        </section>
+
+        <!-- FILTERS -->
+        <section class="filters-section">
+            <div class="container">
+                <div class="filters-row">
+                    <div class="filter-group">
+                        <span class="filter-label">Category</span>
+                        <button
+                            class="filter-btn active"
```

HTMLHint (1.9.2)

[warning] The type attribute must be present on <button> elements.

(button-type-require)